prepline_general/api/general.py

```diff
@@ -278,13 +278,16 @@ def pipeline_api(
 
             # This will raise if the file is encrypted
             pdf.metadata
-        except pypdf.errors.EmptyFileError:
+        except (pypdf.errors.EmptyFileError, pypdf.errors.PdfStreamError):
             raise HTTPException(status_code=400, detail="File does not appear to be a valid PDF")
         except pypdf.errors.FileNotDecryptedError:
             raise HTTPException(
                 status_code=400,
                 detail="File is encrypted. Please decrypt it with password.",
             )
+        except Exception as e:
```

Comment:
this is too broad. other things could have gone wrong

Reply:
Fair point, I figured we'd log the additional errors, but they'll be easier to spot as a new 500. Sentry also found some `PdfReadError`, so I'll add another case above.